MANIFEST.in

```diff
@@ -0,0 +1,20 @@
+graft docs
```

Comment:
I forgot to remove that commit/file. Its supposed to replace the `_get_source_files` in `setup.py`.
I didn't get around to testing it though. Removing it fixes the conda recipe. 

Let me fix that.